config/k8s-etcd.txt

```diff
@@ -0,0 +1,116 @@
+# [This file is part of the zabbix-agent-osso package]
+k8s-etcd
+===========
+
+K8s-etcd monitors an in-cluster etcd setup.
+The status of the etcd is written to a configmap with the following(example)
+pod.
+
+statefulset:
+```
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  labels:
+    run: etcd-status-sender
+    application: patroni
+  name: etcd-status-sender
+  namespace: production-postgres
+spec:
+  selector:
+    matchLabels:
+      run: etcd-status-sender
+      application: patroni
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        run: etcd-status-sender
+        application: patroni
+    spec:
+      serviceAccountName: etcd-status-sender
+      containers:
+      - image: harbor.osso.io/ossobv/etcdctl:3.5.15
+        name: etcd-status-sender
+        env:
+        - name: ETCD_ENDPOINTS
+          value: patroni-etcd-0.patroni-etcd:2379,patroni-etcd-1.patroni-etcd:2379,patroni-etcd-2.patroni-etcd:2379
+        - name: POD_NAMESPACE
+          valueFrom:
+            fieldRef:
+              apiVersion: v1
+              fieldPath: metadata.namespace
+        - name: ETCD_CM_STATUS_NAME
+          value: etcd-status  # customize if needed
```

Comment:
Ik vraag me af of we hier nog iets "zabbix"-igs voor moeten zetten. `zao-etcd-status`? `zabbix-etcd-status`? :shrug: